jbpm-case-mgmt/jbpm-case-mgmt-impl/src/test/java/org/jbpm/casemgmt/impl/CaseServiceImplTest.java

```diff
@@ -876,6 +876,65 @@ public void testCaseWithComments() {
             }
         }
     }
+    
+    @Test
+    public void testCaseWithCommentsPagination() {
```

Comment:
@annabaker after talking to @mswiderski about this decided that it's fine - these methods take QueryContext as param so possible paging is expected. Sorry for the alarming :)